OpenDreamCommon/Bytecode/OpcodeMetadata.cs

```diff
@@ -1,10 +1,10 @@
-namespace DMCompiler.Bytecode;
+namespace OpenDreamShared.Common.Bytecode;
 
 /// <summary>
 /// Custom attribute for declaring <see cref="OpcodeMetadata"/> metadata for individual opcodes
 /// </summary>
 [AttributeUsage(AttributeTargets.Field)]
-internal sealed class OpcodeMetadataAttribute : Attribute {
+public sealed class OpcodeMetadataAttribute : Attribute {
```

Comment:
This change was to allow referencing `OpcodeMetadataAttribute` into this common assembly class from the compiler assembly. Let me know if there's a better approach.